graph-db/serverless/src/utils/cmr/__test__/fetchCollectionPermittedGroups.test.js

```diff
@@ -0,0 +1,183 @@
+import nock from 'nock'
+
+import { fetchCollectionPermittedGroups } from '../fetchCollectionPermittedGroups'
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('fetchCollectionPermittedGroups', () => {
+  test('returns collection permitted groups metadata', async () => {
+    const mockedBody = {
+      items: [
+        {
+          concept_id: 'ACL1376510432-CMR',
+          revision_id: 9,
+          identity_type: 'Catalog Item',
+          acl: {
+            group_permissions: [{ permissions: ['read'], group_id: 'AG1337430038-NSIDC_ECS' },
+              { permissions: ['read'], group_id: 'AG1337430039-NSIDC_ECS' },
+              { permissions: ['read'], group_id: 'AG1337409405-CMR' },
+              { permissions: ['read'], group_id: 'AG1386452130-CMR' },
+              { permissions: ['read'], group_id: 'AG1337409406-CMR' },
+              { permissions: ['read'], user_type: 'registered' },
+              { permissions: ['read'], group_id: 'AG1337409407-CMR' }],
+            catalog_item_identity: [Object],
```

Comment:
This `Object` is just a product of whatever tool this was output to not rendering that deep, this should contain data or just be an empty array.

Reply:
Fixed